app/pages/DateOfBirthPage.scala

```diff
@@ -17,22 +17,29 @@
 package pages
 
 import controllers.routes
-
-import java.time.LocalDate
+import models.registration.responses.{AlreadySubscribedResponse, MatchResponseWithId}
 import models.{NormalMode, UserAnswers}
-import play.api.mvc.Call
 import play.api.libs.json.JsPath
+import play.api.mvc.Call
+
+import java.time.LocalDate
 
 case object DateOfBirthPage extends QuestionPage[LocalDate] {
 
   override def path: JsPath = JsPath \ toString
 
   override def toString: String = "dateOfBirth"
 
-  override protected def nextPageNormalMode(answers: UserAnswers): Call =
-    answers.get(HasNinoPage) match {
-      case Some(true)  => routes.IndexController.onPageLoad()
-      case Some(false) => routes.AddressInUkController.onPageLoad(NormalMode)
-      case None        => routes.JourneyRecoveryController.onPageLoad()
+  override protected def nextPageNormalMode(answers: UserAnswers): Call = answers.get(HasNinoPage) match {
```

Comment:
If you only rename the method above, the HasNinoPage check is not needed. Lets discuss this in a call.

Reply:
As discussed, no change.